### PR TITLE
fixed flex details on flow between devices and menu icon position

### PR DESCRIPTION
**Pull request by rebeckaalsterlind**
-flex on mobile,  -first-page, -second-page, footer

-adjusted width on text for better flow between devices

-fixed menu icon position and removed display: none all devices. 
-menu slide out needs adjusting (for devices)

### CODE REVIEW

File: src/scss/_main.scss [hunk 1]
```diff
@@ -110,11 +111,25 @@ color: $color-citrus;
   padding-bottom: 184px;
 }
 
+.shows-text-wrapper {
+  margin-left: 0px;
+  margin-right: 48px;
+  max-width: 575px;
```
Comment (EmmaHammar) [inline src/scss/_main.scss:117]: bör detta också vara delbart med 4?  ex 576?

File: src/scss/_main.scss [hunk 2]
```diff
@@ -14,28 +14,29 @@ $debugging: true;
...
 .first-page {
-
   padding-top: 40px;
   padding-bottom: 336px;
 }
 
 .intro-text {
   margin: 28px 48px 28px 0;
+  max-width: 575px;
+  border: 1px solid purple;
```
Comment (EmmaHammar) [inline src/scss/_main.scss:39]: kanske kan ta bort denna?

File: src/scss/_ipad-mini.scss [hunk 3]
```diff
@@ -81,12 +81,20 @@ input {
...
+    max-width: 768px;
 }
 
 .shows-wrapper2 {
     margin-top: 128px;
 }
+
+.cards {
+    //margin: 0 5% 0 5%;
+    border: 2px solid red;
```
Comment (EmmaHammar) [inline src/scss/_ipad-mini.scss:96]: ta bort?

File: src/scss/_ipad-mini.scss [hunk 4]
```diff
@@ -81,12 +81,20 @@ input {
 }
 
 .all-shows-wrapper {
-    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    border: 2px solid green;
```
Comment (EmmaHammar) [inline src/scss/_ipad-mini.scss:86]: ta bort?

**Review by EmmaHammar — APPROVED** [review]
Ser super ut, bara att merga!

File: src/scss/_main.scss (hunk 1 above)
Comment (rebeckaalsterlind) [reply src/scss/_main.scss:117]: ja jag antar det

File: src/scss/_ipad-mini.scss (hunk 4 above)
Comment (rebeckaalsterlind) [reply src/scss/_ipad-mini.scss:86]: ta bort border menar du?

**Pull request closed without merging**